### DISCUSSION TITLE
《宝可梦》系列游戏汉化时间表

### DISCUSSION THREAD

**Discussion by giscus-xzonn** [Xzonn/xzonn.github.io#15] [General]
# 《宝可梦》系列游戏汉化时间表

《宝可梦》系列游戏汉化时间表 - 来自Xzonn的小站

https://xzonn.top/posts/Pokemon-Games-Chinese-Localization-History.html

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****diaowinner 评论于 2020-09-05 11:42:58 (UTC+8)*****

sss888还有美版的自己翻译的版本，不过是半官译的

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****乐·learn 评论于 2022-12-10 00:53:24 (UTC+8)*****

2020年11月30日 CKN&DMG&口袋群星SP《精灵宝可梦 水晶版》汉化版V1.0发布  
并在之后的两年内不断发布更新(https://tieba.baidu.com/p/7120594415?pid=136729281610&cid=#136729281610)  
希望可以更新一下水晶版的汉化信息以方便后来人
